slapo/schedule.py

```diff
@@ -519,6 +519,48 @@ def hook_fn(_module, _input):
     def get_module(self, name):
         return dict(self.mod.named_modules())[name]
 
+    def construct_fx_graph(self, subgraph):
+        """Construct a new fx.Graph based on the subgraph extracted from the
+        original graph.
```

Comment:
Maybe saying this function should not be called directly? btw maybe we should rename all internal used functions such as `replace_function` by adding a prefix `_`.